app/test_main.py

```diff
@@ -1,3 +1,44 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+class TestHumanAgeConversion:
+    @pytest.mark.parametrize(
+        "cat_age, dog_age, expected_result",
+        [
+            pytest.param(-1, -1, [0, 0]),
+            pytest.param(0, 0, [0, 0]),
+            pytest.param(14, 14, [0, 0]),
+            pytest.param(15, 15, [1, 1]),
+            pytest.param(23, 23, [1, 1]),
+            pytest.param(24, 24, [2, 2]),
+            pytest.param(27, 27, [2, 2]),
+            pytest.param(28, 28, [3, 2]),
+            pytest.param(100, 100, [21, 17]),
+        ]
+    )
+    def test_get_human_age(self,
+                           cat_age: int,
+                           dog_age: int,
+                           expected_result: list[int]) -> None:
+        assert get_human_age(cat_age, dog_age) == expected_result
+
+    @pytest.mark.parametrize(
+        "cat_age, dog_age, expected_error",
+        [
+            pytest.param(None, None, TypeError,
```

Comment:
No need to add TypeError to params since you will use only one type of error here